### DISCUSSION TITLE
How to build tabby binary for many linux ?

### DISCUSSION THREAD

**Discussion by yanpeiqiong7133** [TabbyML/tabby#3434] [Q&A] [answered]
When I build a tabby binary on centos9 from source code package， it can work correctly in centos，but it can not work in centos7 and other linux os

How can I build a tabby binary which can work in multiple linux operating system's？ Like tabby_x86_64_manylinux2014.zip which is provided by assets link.

**Comment by wsxiaoys** [comment, marked as answer]
Please refer https://github.com/TabbyML/tabby/blob/main/.github/workflows/release.yml on how manylinux binary is built.
